pages.zh/common/ag.md

```diff
@@ -9,11 +9,11 @@
 
 - 在指定目录中寻找内容包含 "foo" 的文件：
```

Comment:
what about this non-translated `foo`?

Reply:
> what about this non-translated `foo`?

I just adjust the parameters in the use case to be consistent. Foo is only a literal value and has no special meaning.